tox.ini

```diff
@@ -202,43 +335,25 @@ commands =
 ; implicit Any due to unfollowed import would result).
   mypyinstalled: mypy --install-types --non-interactive --namespace-packages opentelemetry-api/tests/mypysmoke.py --strict
 
-[testenv:spellcheck]
+[testenv:lint]
```

Comment:
Do we still need this env?

Reply:
No, but adding it to be consistent with [this](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2482).

Reply:
It's used by the semconv generator